props/blaster.h

```diff
@@ -260,9 +266,39 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
     }
   }
 
+  RefPtr<BufferedWavPlayer> tmp;
+  uint32_t len;
+
   void Loop() override {
     PropBase::Loop();
     PollNextAction();
+	  
+    STATE_MACHINE_BEGIN();
+    while (true) {
+      if (auto_firing_) {
+        tmp = GetWavPlayerPlaying(&SFX_auto);
+        // Set the length for WavLen<>
+        if (tmp) {
+          tmp->UpdateSaberBaseSoundInfo();
```

Comment:
This function call only makes sense in SB_* callbacks.
Calling it from Loop() is weird and is likely to cause problems.